src/plugins/bodhi.c

```diff
@@ -271,6 +270,36 @@ static void bodhi_print_errors_from_json(json_object *json)
     return;
 }
 
+/**
+ * Parses only name of rpm package
+ */
+static int parse_pkginfo_name(const char *pkg_full_name, char **name)
```

Comment:
Yes it dies, but it keeps the same behavior as with original hawkey function. When name doesn't match package naming rules, function return EINVAL (invalid input). I just replaced library function with its simplified equivalent. What do you suggest? Should it just log message and continue in cycle?